intl/msg_hash_es.c

```diff
@@ -1272,7 +1272,7 @@ const char *msg_hash_to_str_es(enum msg_hash_enums msg)
       case MENU_ENUM_LABEL_VALUE_CHEEVOS_DESCRIPTION:
          return "Descripción";
       case MENU_ENUM_LABEL_VALUE_CHEEVOS_HARDCORE_MODE_ENABLE:
-         return "Modo Extremo";
+         return "Achievements Modo Extremo";
```

Comment:
I thought about adding `Logros`, but I wouldn't want you guys to yell at me for doing all bad Google Translations too (Chinese, Italian, Spanish, etc) . I opted not to do that. I still can see `Logros` in my clipboard history. :smile: 

I added `Achievements` to stay consistent with other options. If I don't, some people won't know that some lines will need to be translated.